### DISCUSSION TITLE
Always occurring errors on protein-ligand complex

### DISCUSSION THREAD

**Discussion by wonwill** [YAMACS-SML/YAMACS#8] [Q&A] [answered]
Thanks to your previous comments, I am doing running gmx using my protein-ligand complex.
However, I have two more questions about this.

**1. Whenever using amber03 force field, three errors always happen.** So, I have to repeat YAMACS from the beginning.
a) No such file or directory: ligand.mol2, therefore, no ligand.acpype
But after I repeat 1_prep script with default mdp files, it works.
b) No such file or directory: md_0_1.xtc
But after I quit gmx process (kill) and repeat it from 1_prep script with default mdp files, it works.
c) I tried to use the existing mdp files but it failed, due to error "No such file or directory: protein.txt"

**2. When trying to use charmm36 force field, my ligand.pdb disappear after starting 1_System_preparation.**
<img width="232" alt="CleanShot 2023-10-21 at 10 50 04@2x" src="https://github.com/YAMACS-SML/YAMACS/assets/5327542/f8a47722-cdf4-4d37-aa41-1ce8c6465971">

I pasted the file again several times but eventually it failed.

Could you help me out with this errors?

**Comment by YAMACS-SML** [comment, marked as answer]
First question's answer:
It may occurs because there may not present the "ligand.pdb" file in your folder. So to do that, you must need to use the 1st script. You can not directly click on md_run script. You have to run all the four scripts in the sequential manner. If you have already prepared a system from CHARMM-GUI server. Then only you can dirctly click on the "md_run" plugin. Otherwise you must need to run all the four plugins one after another.

Second question's answer:
If i understood correctly, then you did not properly insert the name of the ligand in the protein preparation toolbox window. Do this step one after another:
1. clean your protein and save as protein.pdb file
2. clean your ligand and save as liagnd.pdb file
3. then load your protein.pdb file on yasara window
4. start yamacs. in the attach object section insert "ligand". the continue the process
